README.md

```diff
@@ -85,6 +85,36 @@ res = client.delete(
 print(res)
 ```
 
+# migration tool
```

Comment:
All data stored in Milvus Lite can be easily exported and loaded into other types of Milvus deployment, such as Standalone on Docker, Cluster on K8s, or Zilliz Cloud (fully-managed cloud service for Milvus)